pkg/child/child_test.go

```diff
@@ -201,6 +201,14 @@ func TestEnviron(t *testing.T) {
 			overrideEnv: []string{},
 			exp:         []string{},
 		},
+		{
```

Comment:
I think that we should could guardian changes with workflow executions to at least plan a real terraform change.